Bottom: 1fcb07f3b33db79b7e355cf09a4062adf33a655c
Top:    8831309c9c29f63e271f493d10e6cab77af38130
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:01:18 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
